roles/kubernetes/master/templates/kubeadm-config.v1beta2.yaml.j2

```diff
@@ -195,7 +195,10 @@ apiServer:
 {% endif %}
```

Comment:
Weird that the break line bellow is necessary not to break the build

Reply:
yes, otherwise `event-ttl` line ends up at the end of the previous line